DevelopedCalculator.js

```diff
@@ -1,9 +1,11 @@
 class DevelopedCalculator extends Calculator {
     render() {
         super.render()
-        const history = document.createElement('div');
-        const historyDiv_ = document.createTextNode("History:");
-        const historyDiv = this.hostElm.prepend(historyDiv)
+        const historyDiv = document.createElement('div');
+        const historyDivText = document.createTextNode("History:");
+        historyDiv_.appendChild(historyDivText);
+        this.hostElm.prepend(historyDiv);
+        historyDiv.appendChild(historyDiv_);
```

Comment:
this how you should do it:

```js
// first create a div 
const historyDiv = document.createElement("div");
// add the text to the div
historyDiv.appendChild(document.createTextNode("History:"));
// prepend the historyDiv in the host element 

this.hostElm.prepend(historyDiv);

```
does this make sense?

Reply:
Unfortunately no
Because It is not looks like what you thought me in the Calculator class